flang/runtime/character.h

```diff
@@ -87,9 +87,9 @@ void RTNAME(CharacterPad1)(char *lhs, std::size_t bytes, std::size_t offset);
 
 // Intrinsic functions
 // The result descriptors below are all established by the runtime.
-void RTNAME(Adjustl)(Descriptor &result, const Descriptor &,
+void RTNAME(AdjustL)(Descriptor &result, const Descriptor &,
```

Comment:
https://reviews.llvm.org/D103392 in llvm/main solved this the other way round (changing the names in the .cpp), I think this patch should align with llvm/main here.

Reply:
Thanks -- missed the previous discussion!  

Will push the suggested fix.  

For the record, IMHO it was not intuitive that the naming conventions should change in this particular case.  Not a big deal but this did trip me up as someone new to the code base.

Reply:
Pushed the fix(es).